src/workloads/execution/ColumnstoreIndexes.yml

```diff
@@ -0,0 +1,467 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  This workload compares the query performance with columnar indexes to performances without them.
+
+  The workload consists of the following phases and actors:
+    0. Warm up cache.
+    1. Run queries with columnstore indexes.
+    2. Drop columnstore indexes.
+    3. Warm up cache again for regular scans.
+    4. Run queries without columnstore indexes.
+
+Keywords:
+- columnstore
+- analytics
+
+# Defines the database name.
+Database: &db charts-metrics
+
+# Defines the collection name.
+Collection: &Collection events
+
+DateFieldProjectionQuery: &DateFieldProjectionQuery
+  aggregate: *Collection
+  pipeline:
+    [{"$project": {"_id": 0, "created_at": 1}}]
+  allowDiskUse: true
+  cursor: {batchSize: 2000}
```

Comment:
Can you help me better understand why we are setting the batch size here?

Reply:
AFAIK, the default batch size is 201. Here, I'm trying to increase the batch size so that returning result is more efficient.

Reply:
Actually, are you sure that you are exhausting the cursor? It looks like possibly you are just running the 'aggregate' command which generates a cursor and the first batch, but I don't see any logic that would make sure that that cursor is iterated to completion. Could you check on this or let me know if I'm missing something?